cookbooks/bcpc-hadoop/recipes/journalnode_queries.rb

```diff
@@ -0,0 +1,24 @@
+triggers_sensitivity = '60m'
+
+node.set['bcpc']['hadoop']['graphite']['service_queries']['journalnode'] = {
```

Comment:
`node.set` is being deprecated.

Do you need this data saved in the chef-server?